### PR TITLE
Best practices as-per "Open-source library guidance"

### PR DESCRIPTION
**Pull request by martincostello**
Relates to #402.

Update some of the build properties as per [_"Open-source library guidance"_](https://docs.microsoft.com/en-gb/dotnet/standard/library-guidance/).

## Changes
  1. Use NuGet profile name for the author.
  1. Use 64x64 logo image.
  1. Use commas to separate tags, not semi-colon.
  1. _Prepare_ to strong-name the assembly (blocked by #380).
  1. Fix the _assembly_ version to the major version number with all other components as zero, not the _package_ version number.

### CODE REVIEW

File: version.props [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <Project>
   <PropertyGroup>
+    <AssemblyVersion>7.0.0.0</AssemblyVersion>
```
Comment (slang25) [inline version.props]: It's already got an `AssemblyVersion` of 7.0.0.0, is this so we can manage this separately?

Comment (martincostello) [reply version.props]: This is to pin it to `7.0.0.0` for when `7.0.1` happens, which is why I've left it separate, even though it looks duplicated at the moment.

Comment (slang25) [reply version.props]: Cool, we did use to do this:
https://github.com/justeat/JustSaying/commit/0257f8a68dcc4951fa42708f8aa22a8e3253556c#diff-4b2f402501d23abbede6eac0f47783e4L35

I reverted it cause assembly binding redirects are less of a problem these days, but it likely makes it easier for some consumers still, so worth the effort.

Comment (slang25) [reply version.props]: I think this is another area that needs guidance on, cause Microsoft have stopped doing this for all of their packages now, even ones that target full framework.

Comment (martincostello) [reply version.props]: It is called out in the guidance on strong-naming:

image

Comment (slang25) [reply version.props]: Oh nice, I hadn't seen that

**Pull request merged**